streams/src/test/java/org/apache/kafka/streams/processor/internals/PartitionGroupTest.java

```diff
@@ -104,23 +119,19 @@ public void testTimeTracking() {
         // 2:[2, 4, 6]
         // st: 2
         assertEquals(partition1, info.partition());
-        assertEquals(1L, record.timestamp);
-        assertEquals(5, group.numBuffered());
-        assertEquals(2, group.numBuffered(partition1));
-        assertEquals(3, group.numBuffered(partition2));
-        assertEquals(1L, group.timestamp());
```

Comment:
The style check failed because this method was over 150 lines long.

I've consolidated the checks into verifying the timestamps (record and system) and verifying the number of buffered records (total, partition1, partition2).